pkg/resolver/searchSchema_test.go

```diff
@@ -26,10 +27,28 @@ func Test_SearchSchema_Query(t *testing.T) {
 	}
 }
 
+func Test_SearchSchema_Query_WithFilters(t *testing.T) {
+	value1 := "openshift"
+	searchInput := &model.SearchInput{Filters: []*model.SearchFilter{{Property: "namespace", Values: []*string{&value1}}}}
+	propTypesMock := map[string]string{"namespace": "string"}
+	// Create a SearchSchemaResolver instance with a mock connection pool.
+	resolver, _ := newMockSearchSchema(t, searchInput, rbac.UserData{CsResources: []rbac.Resource{}}, propTypesMock)
+
+	resolver.userData = rbac.UserData{CsResources: []rbac.Resource{}}
+	sql := `SELECT DISTINCT "prop" FROM (SELECT jsonb_object_keys(jsonb_strip_nulls("data")) AS "prop" FROM "search"."resources" WHERE ("data"->'namespace'?('openshift') AND ("cluster" = ANY ('{}'))) LIMIT 100000) AS "schema"`
+	// Execute function
+	resolver.buildSearchSchemaQuery(context.TODO())
+
+	// Verify response
+	if resolver.query != sql {
+		t.Errorf("Expected sql query: %s but got %s", sql, resolver.query)
+	}
```

Comment:
We have a mix, but I prefer to use the assert library when possible.
```suggestion
        assert.Equal(t, sql, resolver.query, "should generate query using filters from input")
```